<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>591</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnAdjust">
        <property name="text">
         <string>局部最大熵算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLinearAdjust">
        <property name="text">
         <string>方差归一化算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGammaAdjust">
        <property name="text">
         <string>二维Gamma函数矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHomomorphicFilter">
        <property name="text">
         <string>同态滤波算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHomoFilter2">
        <property name="text">
         <string>同态滤波（巴特沃斯带阻）矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWallisAdjust">
        <property name="text">
         <string>Wallis滤波器矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNakaAdjust">
        <property name="text">
         <string>Naka-Rushton滤波器矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMaskDodAdjust">
        <property name="text">
         <string>背景剪去矫正</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGlapowAdjust">
        <property name="text">
         <string>Global-Local自适应直方图均衡化矫正算法</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btnHomoFilter3">
        <property name="text">
         <string>同态滤波（高斯型滤波器）矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHomoFilter4">
        <property name="text">
         <string>同态滤波（带阻自动）矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAutoGamma">
        <property name="text">
         <string>自动Gamma矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBkgFitAdjust">
        <property name="text">
         <string> 多项式背景拟合算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGammaFitAdjust">
        <property name="text">
         <string>基于图像梯度的光照拟合算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEstimateBkg">
        <property name="text">
         <string>基于概率的背景估计</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLocalGamma">
        <property name="text">
         <string>局部幂函数矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGumAdjust">
        <property name="text">
         <string>非锐化掩膜矫正算法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMultiAdjust">
        <property name="text">
         <string>多方法叠加测试</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
